src/components/Card/Card.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text } from '../Text';
+import { Heading } from '../Heading';
+import styles from './Card.scss';
+
+export interface Props {
```

Comment:
This should extend usual Div attributes, which should then be passed to the top `div` via `...rest`. This will make sure that e.g. `style` or aria attributes can be passed by the 3rd party user.